libs/src/utils/localStorage.ts

```diff
@@ -1,17 +1,13 @@
+import type { LocalStorage } from '@audius/hedgehog'
```

Comment:
cc @sliptype  might make more sense to leave the custom `LocalStorage` definition in this file instead of importing it from Hedgehog (even though it would be equivalent) since we use it in contexts within libs other than Hedgehog. not a huge deal to me though

Reply:
Yeah I was thinking the same, I think this is fine though. It's just a simple type anyway

Reply:
Yeah I was thinking the same, I think this is fine though. It's just a simple type anyway

Reply:
Yeah that all makes sense to me, will bring it back